hw/ip/otp_ctrl/data/otp_ctrl.hjson

```diff
@@ -193,71 +193,52 @@
       fields: [
         { bits: "0"
           name: "CREATOR_SW_CFG_ERROR"
-          desc: '''
-                Set to 1 if an error occurred in this partition. If set to 1, SW should
-                check the !!ERR_CODE register at the corresponding index to root-cause the error.
-                '''
+          desc: "If set to 1, SW should check the !!ERR_CODE register at the corresponding index."
```

Comment:
I somewhat prefer having the additional sentence ("Set to 1 if an error occurred in this partition.") since it is more declarative about what has happened.  Or was there a reason to remove this?

Reply:
Sure I can add it back in. I intended to shorten the description here a bit since this is repeated for all bits, but I see that it is less declarative if omitted...